### PR TITLE
Generic branding

### PR DESCRIPTION
**Pull request by FireyFly**
This project should only use generic branding everywhere, that could be overridden by the user. This adds that functionality.

### CODE REVIEW

File: packages/next-dashboard/src/components/layout/Header.js [hunk 1]
```diff
@@ -13,38 +13,41 @@ type Props = {
...
+              sidebarActive={sidebarActive}
+              toggleSidebarActive={toggleSidebarActive}
+            />
+          </div>
+          <div className="cell dashboard-header-cell dashboard-header-cell_logo">
+            <Link href="/dashboard">
+              <a className="dashboard-header-logo">
+                <img src={`/images/${logo}`} alt={name || 'Logo'} />
+              </a>
+            </Link>
```
Comment (SimplyLinn) [inline packages/next-dashboard/src/components/layout/Header.js]: While we're at it... could we make homepage and image URL customizeable? :3

(On xvision-dashboard the link href 404's atm)

File: packages/next-dashboard/src/utils/createDashboardHOC.js [hunk 2]
```diff
@@ -98,16 +100,21 @@ export default function createDashboardHOC({
...
   error: errorConf,
-  themes: confThemes,
+  themes: themesConf,
+  branding: brandingConf,
 }: Config): <U: {}, Q: {} = {}>(
   Comp: DashboardComponent<U, Q>,
   needAuth?: boolean,
 ) => NextComponent<WrappedUnified<U, Q>, InitialUnified<Q>> {
-  const themes: Theme[] = confThemes || [
+  const themes: Theme[] = themesConf || [
```
Comment (SimplyLinn) [inline packages/next-dashboard/src/utils/createDashboardHOC.js]: You're calling my naming scheme inconsistent? >.>

Comment (FireyFly) [reply packages/next-dashboard/src/utils/createDashboardHOC.js]: Yes :p

so I decided to fix it up whilst I'm touching this code anyway

File: packages/next-dashboard/src/components/layout/Header.js (hunk 1 above)
Comment (FireyFly) [reply packages/next-dashboard/src/components/layout/Header.js]: Should be sorted now!

**Pull request merged**